Source/Fuse.Controls.Navigation/EdgeNavigator.ScriptClass.uno

```diff
@@ -10,16 +10,16 @@ namespace Fuse.Controls
 		static EdgeNavigator()
 		{
 			ScriptClass.Register(typeof(EdgeNavigator),
-				new ScriptMethod<EdgeNavigator>("dismiss", dismiss, ExecutionThread.MainThread),
-				new ScriptMethod<EdgeNavigator>("open", open, ExecutionThread.MainThread));
+				new ScriptMethod<EdgeNavigator>("dismiss", dismiss),
+				new ScriptMethod<EdgeNavigator>("open", open));
 		}
 
 		/**
 			Closes any open edge panels.
 			
 			@scriptmethod dismiss()
 		*/
-		static void dismiss(Context c, EdgeNavigator e, object[] args)
+		static void dismiss(EdgeNavigator e, object[] args)
 		{
 			if (args.Length != 0)
```

Comment:
This function is actually a no-argument function and should lose the `args` parameter -- assuming we still report an error on calling it.

Reply:
There are a few more such functions, anything doing `if (args.length != 0)`.

Reply:
We don't report errors when dropping the arguments. I tried doing this, but a bunch of tests started failing due to unexpected diagnostics.

Thinking about this a bit more, I'm a bit puzzled why that happened in the first place. Repeating the experiment indicates that I did some error in doing so. So I can clean this up, it seems.

Reply:
I would prefer if we could do behavioral changes like this as a follow-up, though.

Reply:
Sure, if you create an issue to do it afterward. :)

Reply:
https://github.com/fusetools/fuselibs-public/issues/733